File: 229-majority-element-ii/majority-element-ii.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        unordered_map<int,int> mp;
        vector<int> ans;
        for(int num : nums){
            mp[num]++;
        }
        int m = nums.size()/3;
        for(auto n : mp){
            if(n.second > m){
                ans.push_back(n.first);
            }
        }
        return ans;
    }

};

File: 229-majority-element-ii/majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        map = {}
        ans = []
        n = len(nums)
        for num in nums:
            if num not in map:
                map[num] = 1
            else:
                map[num] += 1

        for num in map:
            if map[num] > (n/3):
                ans.append(num)
        return ans